pkg/controller/status/status.go

```diff
@@ -200,7 +200,12 @@ func (c *Controller) merge(existing *configv1.ClusterOperator) *configv1.Cluster
 	if len(c.namespace) > 0 {
 		existing.Status.RelatedObjects = []configv1.ObjectReference{
 			{Resource: "namespaces", Name: c.namespace},
-			{Group: "apps", Resource: "deployments", Namespace: c.namespace, Name: "insights-operator"},
```

Comment:
Well I think we can keep this one...

Reply:
You can keep it if you like, but deployments that live in referenced namespaces [are covered too][1].

[1]: https://github.com/openshift/oc/blame/13f3e4b73df9b0853979e4130adf756784afe96f/pkg/cli/admin/inspect/namespace.go#L18-L30